index.js

```diff
@@ -42,6 +43,13 @@ export default class MapboxAccessibility {
       let { width, height } = this.options;
       const label = feature.properties[this.options.accessibleLabelProperty];
 
+      if(this.geolocatePosition) {
```

Comment:
I'm in favour of keeping the style consistent with the existing code, so `if (` instead of  `if(`. I would have thought eslint would pick this up, so we should tighten eslint I think. @tristen any preference on this?